### DISCUSSION TITLE
Controlling and vacation plugin, overtime payout

### DISCUSSION THREAD

**Discussion by reutherjakob** [kimai/kimai#2447] [Questions & Answers]
Hello, is there a chance to test the controlling and vacation plugin? 

We are looking for a solution:
- see employee overtimes (should be possible with controlling plugin?)
- manage overtime payouts -> reduce overtime 
- monitor open vacation days (should be possible with vacation plugin?)

Thanks in advance. 
Regards,
Jakob

**Comment by kevinpapst** [comment]
http://demo-branch.kimai.org 
Credentials: https://www.kimai.org/demo/

**Reply by reutherjakob** [reply]
Thanks for the quick reply. 
I get the following alert clicking on vacation view. 
![grafik](https://user-images.githubusercontent.com/55136944/111754791-ad5ea680-8898-11eb-9c28-307d8d0551b9.png)

Can we manage overtime payouts somehow?

**Reply by kevinpapst** [reply]
Demo is reinstalled, Vacation works now.
ping @hmr-it-jr for further questions

**Reply by reutherjakob** [reply]
@hmr-it-jr is there a chance to manage overtime payouts with some plugins?
